remoteworkitem/remoteworkitem_test.go

```diff
@@ -0,0 +1,80 @@
+// +build unit
+
+package remoteworkitem
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/almighty/almighty-core/test"
+)
+
+func provideRemoteGithubData() ([]byte, error) {
+	url := "https://api.github.com/repos/almighty/almighty-core/issues/131"
+	response, err := http.Get(url)
```

Comment:
You are correct. This method is called optionally if the example/test/<locally_stored_output>.json is not present or if a flag is passed explicitly for refreshing the xample/test/<locally_stored_output>.json 

Otherwise this is not called.